test/conftest.py

```diff
@@ -143,6 +143,68 @@ def run_js(self, code):
             catch (error) {{ console.log(error.stack); throw error; }}"""
         return self.driver.execute_script(catch)
 
+    def initWebWorker(self):
```

Comment:
```suggestion
    def setup_webworker(self):
```

For PEP8 compliance.  I'm actually not sure why our linting missed this...